massa-node/base_config/config.toml

```
@@ -133,8 +133,10 @@
     max_operations_per_message = 1024
     # max number of endorsements per message
     max_endorsements_per_message = 1024
-    # max milliseconds to wait while sending an event before dropping it
-    max_send_wait = 0
+    # max milliseconds to wait while sending a node event before dropping it
+    max_send_wait_node_event = 5_000
```

I think toml doesn't suppoer "_" separators

```suggestion
    max_send_wait_node_event = 5000
```